src/deepEqual.js

```diff
@@ -20,7 +20,35 @@
  * @return {boolean}
  */
 function deepEqual(a, b) {
-  // write code here
+  if (a === null && b === null) {
+    return true;
+  } else if ((a === null && b !== null) || (a !== null && b === null)) {
+    return false;
+  }
```

Comment:
null === null is true, so you don't need to write such amount of code

Reply:
fixed